src/lib/patterns.js

```diff
@@ -46,6 +46,7 @@ const allPatterns = files.reduce((patterns, file) => {
       title: fileContents.data.title,
       description: fileContents.data.description,
       hero: fileContents.data.hero,
+      draft: fileContents.data.draft,
```

Comment:
Could you add a comment above this section (const/if) saying something like "This only reads front-matter from files, and doesn't inherit any "11tydata.js" files".